pkgs/applications/science/math/primecount/default.nix

```diff
@@ -24,10 +29,10 @@ stdenv.mkDerivation rec {
   ];
 
   meta = with lib; {
-    description = "Fast prime counting function implementations";
     homepage = "https://github.com/kimwalisch/primecount";
+    description = "Fast prime counting function implementations";
     license = licenses.bsd2;
+    inherit (primesieve.meta) maintainers;
     platforms = platforms.unix;
-    maintainers = teams.sage.members;
```

Comment:
I would suggest to combine the maintainers.

Reply:
Sure.
I though it was some error initially.